jobs/service-fabrik-broker/templates/bin/health_check.erb

```diff
@@ -1,13 +1,27 @@
 #!/bin/bash
 
-nc -z $(hostname -I) <%= p('external.port') %>
+# Checking if HA setup needs to be done or not.
+ha_enabled=<%= p('ha_enabled') %>
+if [[ "$ha_enabled" == "true"  ]]; then
```

Comment:
who calls this health check script? If this is being invoked from within keepalived job then its given that ha_enabled right? why these checks then? Doesnt it seem redundant?

Reply:
it is called by route registrar.